CMSgov/easi-app
# [NOREF] Remove references to CEDAR LDAP

## Changes and Description

- Remove, replace, and rename references to CEDAR LDAP (which is no longer used), with appropriate references (most often to Okta, instead)
- Remove .envrc variable that switched between using LDAP and Okta. Now that we're using Okta in every environment, this value is no longer needed.

## How to test this change
- Set up your `.envrc.local` to have an `OKTA_API_TOKEN` value from 1Password.
- Ensure that the app behaves as expected, most commonly when looking up user info. (You can test this most readily on pages like adding TRB attendees)
  - Ensure that this behavior works if you comment out [this check](https://github.com/CMSgov/easi-app/blob/820cac39aed2738d628beaea2c0572cd9a019434/pkg/server/routes.go#L111-L112) (forcing the app to use the _real_ Okta API, instead of a local mock)
  
## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
